src/coalgebras/llistScript.sml

```diff
@@ -2644,16 +2644,75 @@ val LLENGTH_LREPEAT = Q.store_thm(
   `LLENGTH (LREPEAT l) = if NULL l then SOME 0 else NONE`,
   rw[LREPEAT_def])
 
+(* --------------------------------------------------------------------------
+   Case constant, distinctness etc. for TypeBase
+   -------------------------------------------------------------------------- *)
+
+Definition llist_CASE_def:
```

Comment:
If you're going to provide a later `compute` theorem for this constant, the definition should be `nocompute`.

Reply:
check!